package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "webpack-template",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack serve"
+  },
```

Comment:
- [ ] Kindly add `"start": "webpack serve --open"` in the scripts part to be able to run the project using 'npm start` as it will make it easy for you to develop the project and see the changes on the screen.